src/model/framework/manager/AirportManager.ts

```diff
@@ -19,6 +19,16 @@ export class AirportManager implements ISearchBase<IcaoCode> {
         return searchResult.data;
     }
 
+    public async getAirportFrequencies(icao: string): Promise<AirportFrequencyResponse> {
+        const searchResult = await axios.get<AirportFrequencyResponse>(`${autoCompleteBaseUrl}/airport/getFrequencies`, {
```

Comment:
Is there a reason as to why the none 200 error states in this class was never checked? i think we need to handle errors here.

Reply:
Yeah, few places miss the response checks. I will go through all places where api calls are made and check them.

Reply:
Maybe track that in a separate PR. It will have updates related to other classes as well.